include/net/ieee802154_radio.h

```diff
@@ -171,8 +172,8 @@ enum ieee802154_config_type {
 	/** Configure a radio reception slot */
```

Comment:
I think these options require more detailed comment describing that `RX_SLOT` can be used for any scheduler reception like Zigbee GP device, CSL, TSCH.

To implement CSL this option must be used together with the other two options. It should be explained when, why, and in what order these options must be set to maintain CSL receiver active.

Reply:
Is the `rx_slot` struct as is compatible with all the mentioned protocols?

Reply:
I believe it is, but I have not tried to implement any. I might have overseen some missing details.